packages/create-cloudflare/src/helpers/args.ts

```diff
@@ -54,7 +53,12 @@ export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
 		})
 		.option("existing-script", {
 			type: "string",
-			hidden: templateMap["pre-existing"].hidden,
+			hidden: true,
+		})
+		.option("template", {
+			type: "string",
+			description:
+				"The url of a git repository containing the C3 template to run",
```

Comment:
Can we tweak the description so to make clear that a path within a repo can also be specified?

Something like:
```suggestion
				"The url to a git repository (with optionally a directory path) containing the C3 template to use",
```